apis/Google.Devtools.Aspnet/Google.Devtools.Aspnet/CloudErrorReportingExceptionLogger.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Devtools.Clouderrorreporting.V1Beta1;
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http.ExceptionHandling;
+
+namespace Google.Devtools.Aspnet
+{
+    /// <summary>
+    ///  Google Cloud Error Reporting <see cref="ExceptionLogger"/>.
+    /// </summary>
+    /// 
+    /// <example>
+    /// public static void Register(HttpConfiguration config)
+    /// {
+    ///   // Add a catch all for the uncaught exceptions.
+    ///   config.Services.Add(typeof(IExceptionLogger),
+    ///       CloudErrorReportingExceptionLogger.Create());
+    /// }
+    /// </example>
+    /// 
+    /// <remarks>
+    /// Reports unhandled exceptions to Google Cloud Error Reporting.
+    /// Docs: https://cloud.google.com/error-reporting/docs/
+    /// </remarks>
+    public sealed class CloudErrorReportingExceptionLogger : ExceptionLogger
+    {
+        // The formated Google Cloud Platform project id.
+        private readonly string _projectId;
```

Comment:
At that point, I'd call it `_projectName` or `_projectResourceName`.
See https://googlecloudplatform.github.io/google-cloud-dotnet/docs/resource-names.html for nomenclature.